CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.6.2] - 2024-SEP-09
```

Comment:
Version number is fine for now but I think in the future we need to follow the standards more:
-the last digit should be for a patch, aka a fix to previous changes
-any new backwards compatible feature should update the middle number
-any non-backwards compatible update the first number.

So this should be 1.7.0, correct me if I'm wrong.